com.alientrap.nexuiz-classic.yaml

```diff
@@ -0,0 +1,75 @@
+app-id: com.alientrap.nexuiz-classic
+runtime: org.freedesktop.Platform
+runtime-version: "19.08"
+sdk: org.freedesktop.Sdk
+command: launcher.sh
+finish-args:
+  - --device=dri
+  - --socket=pulseaudio
+  - --socket=wayland
```

Comment:
Does it actually work on Wayland?

Reply:
No, only through XWayland. I've removed it.